index.html

```diff
@@ -63,14 +63,306 @@ <h1 class="hero-title">Want anything to be easy with LaslesVPN.</h1>
+              “I like it because I like to travel far and still can connect with
+              high speed.”.
+            </p>
+          </li>
+          <li tabindex="0" class="testimonials-box">
+            <div class="d-flex justify-content-between align-items-center">
+              <img src="images/testimonials-img3.jpg" alt="" />
+              <p class="flex-direction-column d-flex">
+                <span class="title-3">Kim Young Jou</span>
+                <span class="text-sm">Seoul, South Korea</span>
+              </p>
+              <p>
+                <span>4.5</span>
+                <span class="icon-star-filled testimonial-star"></span>
+              </p>
+            </div>
+            <p class="plain-text-black">
+              “This is very unusual for my business that currently requires a
+              virtual private network that has high security.”.
+            </p>
+          </li>
+        </ul>
+        <div class="d-flex justify-content-between align-items-center">
+          <div class="d-flex justify-content-between">
+            <div class="scroll-bullet"></div>
+            <div class="scroll-bullet"></div>
+            <div class="scroll-bullet"></div>
+            <div class="scroll-bullet"></div>
+          </div>
+          <div>
+            <button class="button-circle">
+              <span class="icon-button-left"></span>
+            </button>
+            <button class="button-circle">
+              <span class="icon-button-right"></span>
+            </button>
+          </div>
+        </div>
+      </section>
+      <section class="subscribe-section container">
+        <div class="subscribe-wraper">
+          <div class="subscribe-hero-box">
+            <h2 class="title-2">Subscribe Now for Get Special Features!</h2>
+            <p class="plain-text">Let's subscribe with us and find the fun.</p>
+          </div>
+          <div>
+            <button class="button-secondary">Subscribe Now</button>
+          </div>
+        </div>
+      </section>
     </main>
-    <footer></footer>
+    <footer>
+      <section class="container footer-section">
+        <div class="footer-subscribe-content">
+          <img src="images/logo.svg" alt="" />
+          <p class="plain-text">
+            <b>LaslesVPN</b> is a private virtual network that has unique
+            features and has high security.
+          </p>
+          <ul class="d-flex">
+            <li>
+              <a class="social-networks" href=""
+                ><span class="icon-facesbook"></span
+              ></a>
+            </li>
+            <li>
+              <a class="social-networks" href=""
+                ><span class="icon-twitter"></span
+              ></a>
+            </li>
+            <li>
+              <a class="social-networks" href=""
+                ><span class="icon-instagram"></span
+              ></a>
+            </li>
+          </ul>
+          <p class="copyright">©2020LaslesVPN</p>
```

Comment:
Use special character for copyright. https://www.teachucomp.com/special-characters-in-html-tutorial/